### PR TITLE
Issue #504

### PR DESCRIPTION
**Pull request by 23mwest**
Sorry this took a while

### CODE REVIEW

**Comment by whyboris** [comment]
Code looks awesome! I'll test it out before merging ✅ 
I hope to finally get back to VHA development this or next month 🤞 -- have been busy with house stuff 😅

**Comment by 23mwest** [comment]
Awesome, congrats on the house stuff! 👍

File: src/app/components/home.component.ts [hunk 1]
```diff
@@ -1695,13 +1698,13 @@ export class HomeComponent implements OnInit, AfterViewInit {
       clipSnippets: 5,
       extractClips: false,
       futureHubName: '',
-      isFixedNumberOfScreenshots: true,
-      screenshotSizeForImport: 288, // default
+      isFixedNumberOfScreenshots: this.wizard.isFixedNumberOfScreenshots ?? true,
```
Comment (whyboris) [inline src/app/components/home.component.ts:1701]: I'm loving the [nullish-coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here 👍

File: src/app/components/home.component.ts [hunk 2]
```diff
@@ -768,6 +768,9 @@ export class HomeComponent implements OnInit, AfterViewInit {
       if (settingsObject.remoteSettings) {
         this.remoteSettings = settingsObject.remoteSettings;
       }
+      if (settingsObject.wizardOptions) {
```
Comment (whyboris) [inline src/app/components/home.component.ts:771]: The interface `SettingsObject` doesn't include `wizardOptions` so this conditional would never evaluate to true.

But I'll merge this PR and just add the wizard options to the settings object when it's written to disk (to persist the user's preference) 👌

File: src/app/components/home.component.ts [hunk 3]
```diff
@@ -1911,6 +1914,7 @@ export class HomeComponent implements OnInit, AfterViewInit {
       shortcuts: this.shortcutService.keyToActionMap,
       vhaFileHistory: this.vhaFileHistory,
       windowSizeAndPosition: undefined, // is added in `cose-window` in `main.ts`
+      wizardOptions: this.wizard
```
Comment (whyboris) [inline src/app/components/home.component.ts:1917]: 😅 ah I see it! You're saving it 🎉

File: src/app/components/home.component.ts [hunk 4]
```diff
@@ -768,6 +768,9 @@ export class HomeComponent implements OnInit, AfterViewInit {
       if (settingsObject.remoteSettings) {
         this.remoteSettings = settingsObject.remoteSettings;
       }
+      if (settingsObject.wizardOptions) {
+        this.wizard = settingsObject.wizardOptions;
```
Comment (whyboris) [inline src/app/components/home.component.ts:772]: After merging I'll add a few lines - maybe resetting the `wizardOptions` property `showWizard` 👍

**Comment by whyboris** [comment]
🙇 Thank you very much @23mwest for your contribution 🎉 
I might add a line or two to catch a _possible_ weird edge case (though I'm unsure if it ever would happen -- the `showWizard` property might be `true` when a user exits - just don't want the app to get stuck in a weird state -- so I'll test a thing or two, even though I already merged it).

One minor addition I'll add is to add the `wizard` property to the _SavedObject_ interface ✅
